### PR TITLE
Adding myself to the dev list

### PR DESCRIPTION
**Pull request by chapmajs**
Thanks!

### CODE REVIEW

**Comment by chapmajs** [comment]
Heh, assuming that's a random fail...

File: ShiroCasGrailsPlugin.groovy [hunk 1]
```diff
@@ -19,6 +19,7 @@ class ShiroCasGrailsPlugin {
         [name: "Ford Guo", email: "agile.guo@gmail.com"],
         [name: "Gavin Hogan", email: "gavin@commercehub.com"],
         [name: "Curt Hostetter", email: "hostetcl@gmail.com"],
+        [name: "Jonathan Chapman", email: "jonathan@enablelabs.com"]
```
Comment (davidmc24) [inline ShiroCasGrailsPlugin.groovy]: Trailing comma, please.

**Comment by chapmajs** [comment]
Done!

**Comment by davidmc24** [comment]
Thanks!

**Pull request merged**